tools/sloths/src/stores/pagination.ts

```diff
@@ -0,0 +1,31 @@
+import { PAGINATION_OPTIONS } from '@/common/const';
+import { defineStore } from 'pinia';
+
+const usePagination = defineStore({
+  id: 'pagination',
+
+  state: () => ({
+    perPage: PAGINATION_OPTIONS[0],
+    currPage: 1,
+  }),
+
+  actions: {
+    getPerPage(): number {
+      return this.perPage;
+    },
+
+    setPerPage(n: number) {
+      this.perPage = +n;
```

Comment:
If `n` is a number, what is the purpose of `+n` opeartion?

Reply:
done
del unused type conversion